pyvrp/Population.py

```diff
@@ -351,6 +357,17 @@ def select(self) -> Tuple[Individual, Individual]:
 
         return first, second
 
+    def restart(self):
```

Comment:
Can we add a test that after calling `restart()`, the population is of size `min_pop_size` and consists of new individuals?

Reply:
Can you check the new test?

Reply:
This works for me. Could perhaps be made a bit nicer if we add a `__contains__` dunder to `SubPopulation`, but that's not really important right now.